server/dash/design.py

```diff
@@ -47,6 +47,24 @@
 }
 
 
+# TODO: Fixing multiple streetlights and waterwaste color
```

Comment:
Can you open an issue for this, and reference it here?

Reply:
Resolve with #1223